src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -326,24 +326,33 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
                 <p class="margin-top">
                     Updating the mod list from Thunderstore might solve this issue.
 
-                    <span v-if="$store.state.tsMods.modsLastUpdated">
-                        The mod list was last updated on {{ valueToReadableDate($store.state.tsMods.modsLastUpdated) }}.
-                    </span>
-
-                    <br />
-
-                    <span v-if="$store.state.tsMods.isThunderstoreModListUpdateInProgress">
-                        {{ $store.state.tsMods.thunderstoreModListUpdateStatus }}
-                    </span>
-                    <span v-else-if="$store.state.tsMods.thunderstoreModListUpdateError">
-                        Error updating the mod list:
-                        {{ $store.state.tsMods.thunderstoreModListUpdateError.message }}.
-                        <a @click="$store.dispatch('tsMods/syncPackageList')">Retry</a>?
-                    </span>
-                    <span v-else>
-                        Would you like to
-                        <a @click="$store.dispatch('tsMods/syncPackageList')">update now</a>?
-                    </span>
+                    <div v-if="$store.getters['download/activeDownloadCount'] > 0">
+                        <span>
+                            However, the mod list can't be updated while the are mod downloads in progress.
+                            Please wait for the downloads to finish before continuing.
```

Comment:
This part will probably be something like "The mod list will try to refresh after the downloads are finished.", or be completely refactored in the future.

Reply:
I'd think it will make sense to handle the update before this box is rendered, perhaps in a separate step before the preview. Then in this step we'll either say that some mods can't be imported, or list all the mods normally if they're all known by the manager.